packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -57,11 +63,13 @@
 
   .help-button {
     margin: 15px;
+    position: static;
     text-align: right;
 
     @media screen and (min-width: $terra-medium-breakpoint) {
       bottom: 15px;
       margin: 0;
+      padding: 0;
```

Comment:
this shouldn't be needed if the component doesn't have padding outside of the media query

Reply:
Will remove that. Used padding instead of margin before so had to set padding 0 for mobile, not needed now.